### PR TITLE
Fix: separate map TLS procedure section

### PR DESCRIPTION
**Pull request by jiliangWANG**
## Description

- separate the secure map with TLS procedure into a page

- fix a section reference

## Target serie

- [X] 20.04.x
- [X] 20.10.x (master)

### CODE REVIEW

**Review by chgautier — APPROVED** [review]

**Review by cgagnaire — COMMENTED** [review]
Not a big fan of the header/numbered list mix.
Not sure it should be used (don't see it in https://guides.github.com/features/mastering-markdown/)
And not sure it will work good with url/links.
What was the point of adding numbered list ?

**Comment by jiliangWANG** [comment]
> Not a big fan of the header/numbered list mix.
> Not sure it should be used (don't see it in https://guides.github.com/features/mastering-markdown/)
> And not sure it will work good with url/links.
> What was the point of adding numbered list ?

Because these "sections" in the "Procedure" chapitre are not certainly related.  And just separate them with font size can be misleading for user.  
In previous document (19.10), these sections are in separated pages.  I don not how to put them as separated pages inside "Procedure" chapitre.

**Review by tuntoja — COMMENTED** [review]
While numbered looks better and easier to read, it gives the false feeling that all of them have to be done in order, but actually there is not need to do everything in this order.
As Colin suggested, it could be split into more specialized pages. Here for instance, in this page there are these big topics
* maintenance (backup / migration / configuration)
* hardening (TLS and security / ports)
* monitoring (monitoring the centreon map server)

**Comment by chgautier** [comment]
The procedures don't have to be numbered but there should be a clear distinction between them. The different procedures are standalone and the broken reference needed to be fixed for better comprehension.

File: en/graph-views/procedures.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ For the requirement of changing service's port, refer to :ref:`change_server_por
 > Don't forget to modify the URL on Centreon side in 
 > **Administration** > **Extensions** > **Map** > **Options** => **Map server address** 
 
-## Broker configuration
+## 5. Broker configuration
```
[the remark this answers is not in the document]
Comment (jiliangWANG) [reply en/graph-views/procedures.md]: Changed

**Comment by cgagnaire** [comment]
I reworked the layout for this secure part.
And I cleaned all files to make it real GitHub markdown.

**Pull request merged**